django_nitro_mailer/tasks.py

```diff
@@ -20,36 +21,36 @@ def throttle_email_delivery() -> None:
         time.sleep(throttle_delay / 1000)
 
 
+def send_email_message(email_data: EmailMessage, connection) -> bool:
+    try:
+        if email_data:
```

Comment:
I tutaj typowanie powinno Ci było pomóc, bo w `email_data` zawsze przyjmujesz `EmailMessage`, więc nie spodziewasz się, że `if email_data` kiedykolwiek nie będzie prawdziwe. Dlatego wynieś proszę stąd ten warunek i sprawdzaj to przed wywołaniem tej funkcji w `send_emails`.